core/src/com/agateau/pixelwheels/racescreen/Helicopter.java

```diff
@@ -180,6 +180,9 @@ public void reset() {}
     @Override
     public void dispose() {
         sPool.free(this);
+        if (mSoundPlayer != null) {
```

Comment:
Makes sense, thanks! I had a few players reporting this bug but could never reproduce it.

Can you just move this code so that the call to `sPool.free(this)` comes at the end of the function? I'd rather avoid calling code after the pool has been freed.

Reply:
Also, can you add a changelog entry to please the CI?

Pixel Wheels' changelog is managed with [Changie](https://changie.dev/).

Reply:
updated.